src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import { Status } from '../types/Status';
+
+type Props = {
+  setStatus: (text: string) => void,
+  status: string,
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  setStatus,
+  status,
+}) => {
+  const handleAllButton = () => {
+    setStatus(Status.ALL);
+  };
+
+  const handleActiveButton = () => {
+    setStatus(Status.ACTIVE);
+  };
+
+  const handleCompletedButton = () => {
+    setStatus(Status.COMPLETED);
+  };
```

Comment:
instead three funcion you can make one function factory and call with proper field for each button

```suggestion
  const handleFilter = (filter: Status) => () => {
    setStatus(filter);   
  };

```